jscomp/syntax/ast_exp_apply.ml

```diff
@@ -111,6 +111,8 @@ let app_exp_mapper
            { pexp_desc = Pexp_apply(fn, (Ast_compatible.no_label, new_obj_arg) :: args);
              pexp_attributes = [];
              pexp_loc = pexp_loc}
+         | {pexp_desc = Pexp_variant(label,None); pexp_loc; pexp_attributes} -> 
```

Comment:
I'm not very familiar with PPX, is `Pexp_variant` referring to polymorphic variants? Also I think you will want to add an example of this new case to the comment above (L105).

Reply:
yes, it is referring to polymorphic variants. Comment was added, thanks :)